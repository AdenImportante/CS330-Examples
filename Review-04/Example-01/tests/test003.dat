#! /bin/sh

# Test entire program with only Tool input

cat <<EOF >test003.in
Tool Shovel Gold 20 3 Unbreaking 2
Tool Pickaxe Stone 10 3 Efficiency 2
EOF

# Strip all spaces and newlines
# convert to all lowercase
./storage test003.in 4 | sed 's/[ ]//g' | grep -vE "^$" | tr '[:upper:]' '[:lower:]'