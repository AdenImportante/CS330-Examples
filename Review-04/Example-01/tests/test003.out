processinglog:
(s)shovel
(s)pickaxe
playerstoragesummary:
-used50%of4slots
nme:shovel
dur:20
spd:3
mtl:gold
mdr:unbreaking(lvl2)
nme:pickaxe
dur:10
spd:3
mtl:stone
mdr:efficiency(lvl2)
